apps/glusterfs/app_volume.go

```diff
@@ -246,8 +246,22 @@ func (a *App) VolumeDelete(w http.ResponseWriter, r *http.Request) {
 			return err
 		}
 
-		return nil
+		if !volume.Info.Block {
```

Comment:
@lpabon the latest copy of the design doc in the PR can be seen under https://github.com/obnoxxx/gluster-kubernetes/blob/gluster-block-design/docs/design/gluster-block-provisioning.md